docs/WindowsBuild.md

```diff
@@ -1,408 +1,78 @@
-  -D CMAKE_Swift_COMPILER=S:/b/1/bin/swiftc.exe ^
-  -D dispatch_DIR=S:\b\2\cmake\modules ^
-  -D Foundation_DIR=S:\b\3\cmake\modules ^
-  -D XCTest_DIR=S:\b\4\cmake\modules ^
-  -G Ninja ^
-  -S S:\swift-argument-parser
+> [!IMPORTANT]
+> Please make sure to consider the security risks listed [here](https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/create-symbolic-links).
 
-ninja -C S:\b\8
-```
+## Cloning the repositories
 
-## Build swift-driver
+- Open a regular (non admin, non Visual Studio) Powershell shell.
+- Create a directory named S at the root of the C:\ drive: `mkdir C:\S`.
+- Navigate to that directory: `cd C:\S`.
+- Clone the Swift repository: `git clone https://github.com/swiftlang/swift.git`.
+- Switch to the branch you want to build: `git -C swift checkout release/6.2`.
+- Clone the rest of the repositories: `.\swift\utils\update-checkout --scheme release/6.2 --reset-to-remote --skip-repository swift`.
 
-```cmd
-cmake -B S:\b\9 ^
-  -D BUILD_SHARED_LIBS=YES ^
-  -D CMAKE_BUILD_TYPE=Release ^
-  -D CMAKE_INSTALL_PREFIX=C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr ^
-  -D CMAKE_MT=mt ^
-  -D CMAKE_Swift_COMPILER=S:/b/1/bin/swiftc.exe ^
-  -D dispatch_DIR=S:\b\2\cmake\modules ^
-  -D Foundation_DIR=S:\b\3\cmake\modules ^
-  -D TSC_DIR=S:\b\5\cmake\modules ^
-  -D LLBuild_DIR=S:\b\6\cmake\modules ^
-  -D Yams_DIR=S:\b\7\cmake\modules ^
-  -D ArgumentParser_DIR=S:\b\8\cmake\modules ^
-  -G Ninja ^
-  -S S:\swift-driver
-
-ninja -C S:\b\9
-```
-
-## Build swift-package-manager
-
-```cmd
-cmake -B S:\b\10 ^
-  -D BUILD_SHARED_LIBS=YES ^
-  -D CMAKE_BUILD_TYPE=Release ^
-  -D CMAKE_C_COMPILER=S:/b/1/bin/clang-cl.exe ^
-  -D CMAKE_INSTALL_PREFIX=C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr ^
-  -D CMAKE_MT=mt ^
-  -D CMAKE_Swift_COMPILER=S:/b/1/bin/swiftc.exe ^
-  -D dispatch_DIR=S:\b\2\cmake\modules ^
-  -D Foundation_DIR=S:\b\3\cmake\modules ^
-  -D TSC_DIR=S:\b\5\cmake\modules ^
-  -D LLBuild_DIR=S:\b\6\cmake\modules ^
-  -D Yams_DIR=S:\b\7\cmake\modules ^
-  -D ArgumentParser_DIR=S:\b\8\cmake\modules ^
-  -D SwiftDriver_DIR=S:\b\9\cmake\modules ^
-  -G Ninja ^
-  -S S:\swift-package-manager
-
-ninja -C S:\b\10
-```
-
-Indicate to swift-package-manager where to find the Package Description before installation:
-```cmd
-set SWIFTPM_PD_LIBS=S:\b\10\pm
-```
-
-## Install the Swift toolchain on Windows
-
-- Run ninja install:
-
-```cmd
-ninja -C S:\b\1 install
-```
+## Starting the build
 
-- Add the Swift on Windows binaries path (`C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin`) to the `PATH` environment variable.
+- Set the `SKIP_UPDATE_CHECKOUT` environment variable to `true` to avoid cloning the repositories again: `$env:SKIP_UPDATE_CHECKOUT="true"`.
+- Start the build: `.\swift\utils\build-windows-toolchain.bat`. This will build the toolchain, its installer and run all the test suite.
```

Comment:
I think that we should document `build.cmd` as the tool that developers should be interacting with, not `build-windows-toolchain.bat`. The latter is specifically for the CI systems.

Reply:
Switched to `build.cmd`.